index.test-d.ts

```diff
@@ -6,38 +6,25 @@ import FirstChunkStream = require('.');
 
 const options: FirstChunkStream.Options = {chunkLength: 1};
 
-expectError(new FirstChunkStream());
 expectError(new FirstChunkStream({}, () => {}));
 
-const firstChunkStream = new FirstChunkStream(
-	{chunkLength: 7},
-	(error, chunk, encoding, callback) => {
+const firstChunkStream = new FirstChunkStream({chunkLength: 7}, async (error, chunk, encoding) => {
```

Comment:
Since we're doing breaking changes anyway, what do you think of renaming `chunkLength` to `chunkSize`? I feel like that would be clearer.

Reply:
IMO, just `length` makes most sense.

Another possible option is just removing it and making function return `give me more` until it's satisfied, but that's another major rework.

Reply:
> IMO, just length makes most sense.

I disagree. I've been thinking about this more and `chunkSize` definitely makes more sense. `length` is only an array term. Size makes much more sense when talking about a chunk of data. For example, the Node.js docs also uses the wording `size` when talking about the amount of data to read, like `readable.read([size])`.

> Another possible option is just removing it and making function return give me more until it's satisfied, but that's another major rework.

That sounds like a separate more powerful feature. For my use-cases, I know the size upfront.

Reply:
No, my point was to use just `size`/`length` without `chunk`. Regarding `size` vs `length`, size sounds better but practice tells me first thing I'd use after a break in usage of lib would be `length`, and then I'd go straight to reading docs at which point the name doesn't matter anymore.

Reply:
With just `size`/`length` it's IMHO not clear enough that it's only about the chunk and not the whole stream.

Reply:
Well, just my suggestion based on how I use stuff(which might be much less important for people who don't disable the slow IDE features like I do, so probably have a popup about type). In the context of using `FirstChunkStream` it's pretty obvious what it means to IMO.
Anyhow, speaking about `length` vs `size`, as I said, size does hint on non-guaranteedness, but I don't think it's really important in that case, though if you think it is there is obviously no problem changing it.